client/components/cards/cardDetails.js

```diff
@@ -152,11 +152,15 @@ Template.moveCardPopup.events({
 });
 
 Template.cardMorePopup.events({
-  'click .js-delete': Popup.afterConfirm('cardDelete', () => {
-    Popup.close();
-    Cards.remove(this._id);
-    Utils.goBoardId(this.board()._id);
-  }),
+  'click .js-delete'(event, tpl) {
```

Comment:
I generally use `evt` here. Could you change it just to be consistent with the rest of the code base?